1-Draft/RFC0008-Export-Import-Json.md

```diff
@@ -0,0 +1,33 @@
+---
+RFC: 0008
+Author: Doug Finke
+Status: Draft
+Version: 1.0
+Area: Export/Import JSON files 
+---
+
+# Implement Export-Json and Export-Json 
+
+Provide cmdlets for importing and exporting Json files (same as with the CSV format). This simplifies the command line interaction, reduces the amount of code needed to be written by users and the user needs to know fewer commands to interact with Json and the file system.  
+
+## Motivation
+
+Adding these cmdlets makes it far simpler to interact with Json and files.
+
+`gc .\test.json | ConvertFrom-Json`
+
+becomes
+
+`Import-Json .\test.json`
+
+Plus, creating an alias and using the pipeline, you can process all the json files in a directory.
+
+`dir *.json | ipjson`
+
+For the `Export-Json` similar benefits can be seen by adding this convenience cmdlet. 
```

Comment:
How about enhancing `ConvertFrom-Json` and `ConvertTo-Json` to support read from and write to files?